test_game.py

```
@@ -11,7 +11,10 @@
 
 # Add the current directory to the path so we can import game
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
-tests conlsoe
+domeanexample()
```

Ruff (0.12.2)

Undefined name `domeanexample`

(F821)